### PR TITLE
doc(variables) Update variables.md

### PR DESCRIPTION
**Pull request by DumitruCorini**
- Corrected grammar mistakes
- Removed some phrases that repeat what was said in previous phrases or contradictions like "String and JSON are initialization types." (We said before that they were static type variables, not initialization type variables)
- Refactor some of the phrases

### CODE REVIEW

File: md/variables.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Variables
 
-**Variables** is a key concept of the UI Designer. They define page, layout, form, and fragment behavior, they are used to fetch external resources, to manipulate and process information, and much more. It is important to understand how to define and use variables in those artifacts, and how the data contained in the variables works when combined with widget properties. Unless otherwise stated, we will use "page" as the generic for page, layout, form, and fragment.
+**Variables** are a key concept in the UI Designer. They define how pages, layouts, forms and fragments behave. They are used to fetch external resources, to manipulate and process information, and much more. It is important to understand how to define and use variables in artifacts, and how the data contained in the variables works when combined with widget properties. Unless otherwise stated, we will use "page" as the generic term for page, layout, form, and fragment.
```
Comment (educhastenier) [inline md/variables.md:5]: :warning: The description is missing

File: md/variables.md [hunk 2]
```diff
@@ -50,11 +50,11 @@ An expression often relies on other variables as dependencies.
 **Warning:** Every time one of these variables changes, the whole JavaScript expression is re-evaluated and the previous value is overwritten.   
 :::
 For example, create a `login` expression variable: `return $data.firstname.toLowercase() + '-' + $data.lastname.toLowercase()`. Its dependencies are the two variables `firstname` and `lastname`.   
-Create two input widgets "First name" and "Last name" bound the two variables, and a text widget "Login" to display the result of the expression. When the user fills out the two input fields, the expression is updated. If the login value is manually edited before the user fills out the fields, then its value is overwritten.
+Create two input widgets "First name" and "Last name" and bind the values to two variables `firstname` and `lastname`.Add a text widget "Login" to display the result of the `login` expression. When the user fills out the two input fields, the expression is updated. If the login value is manually edited before the user fills out the fields, then its value is overwritten.
```
Comment (educhastenier) [inline md/variables.md]:
```suggestion
Create two input widgets "First name" and "Last name" and bind the values to two variables `firstname` and `lastname` .Add a text widget "Login" to display the result of the `login` expression. When the user fills out the two input fields, the expression is updated. If the login value is manually edited before the user fills out the fields, then its value is overwritten.
```

File: md/variables.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Variables
```
Comment (educhastenier) [inline md/variables.md]:
```suggestion
# Variables in the UI Designer
```

**Review by educhastenier — CHANGES_REQUESTED** [review]
The goal of the initiative is to add a description

File: md/variables.md (hunk 3 above)
Comment (DumitruCorini) [reply md/variables.md]: done

File: md/variables.md (hunk 2 above)
Comment (DumitruCorini) [reply md/variables.md]: done

File: md/variables.md (hunk 1 above)
Comment (DumitruCorini) [reply md/variables.md:5]: Added a description

**Review by educhastenier — APPROVED** [review]

**Pull request merged**